core/chains/evm/config/toml/defaults/BOB_Mainnet.toml

```diff
@@ -0,0 +1,27 @@
+ChainID = '60808'
+# OP stack https://docs.gobob.xyz/learn/introduction/op-stack
+ChainType = 'optimismBedrock'
+# finality_depth was: ~850
+FinalityDepth = 900
+# block_time was:  2s
+LogPollInterval = '5s'
+
+# finality_depth * block_time / 60 secs = ~30  min (finality time)
+NoNewFinalizedHeadsThreshold = '35m'
```

Comment:
no that I'm aware of. 
This were significantly bumped bc Worldchain, Unichain and Berachain (can't fully remember the other 2) were throwing a really high number of errors because of this.

Reply:
The easier way is check the questionnaire or track the finalityBlock rate & the timing between the successful blocks. I thought we had a test, but just confirmed it only gives the depth. I think it might make sense to add it to the property tests now

Reply:
@simsonraj / @friedemannf Given a constant block rate, is there a reason we can't interpolate the finality time based on the finality depth? Keeping in mind that finality depth was calculated by averaging the delta between the latest block and the latest finalized block. Let me know if I'm missing any constraints that would impact this approach.
Also, we're optimizing for reducing the error logs so we're using the max finality depth measured (not the min).

Reply:
This is particularly tricky for L2s, even if not for L1 chains ( only assuming here)
Because the Batches happen at a different rate & have to be rolled up & committed/proved on L1, so it will have its own rate separate from the blockrate. We have seen for some chains, the batch posted to L1 goes upto many hours for security measures or other reasons even when the blocks are still being produced in L2. for example Linea with 8-32 hours. 
As Friedemann mentioned in the thread, we will have to understand the finality rate, but I believe the values are not too far off

Reply:
@simsonraj After reviewing all the chains and batch calculations, I noticed an additional factor that could improve the accuracy of the equation: the proportion of blocks added to each batch, or the `batching/finalization percentage`. For instance, on Unichain, only around 30% of generated blocks are included in the latest batch. By incorporating this factor, the equation becomes:

`finality_depth * batching_finalization_percentage * block_time / 60 secs`

This adjustment has given us more precise results for Unichain. For other chains, the numbers are already fairly close.